### PR TITLE
Prevent Multiple Stdlibs From Being Loaded By Default

### PR DESCRIPTION
**Pull request by chrisnevers**
Add `unsafeAllowMultipleStdlibs: () -> null` to allow users to explicitly allow multiple stdlib usage

### CODE REVIEW

File: docs-src/ref-frontends-js.scrbl [hunk 1]
```diff
@@ -115,6 +115,19 @@ In such scenarios, we recommend that you call this function like so:
 You may instead pass in the string @litchar{'ETH'} or the string @litchar{'ALGO'}
 to select the desired stdlib directly.
 
+By default, this method allows a user to load a single standard library.
+That is, this method may not be called multiple times, with the same or varying @tech{connectors}.
```
Comment (jeapostrophe) [inline docs-src/ref-frontends-js.scrbl:119]: Can you make it work on the same connector a second time?

File: examples/maybe-send/index.mjs [hunk 2]
```diff
@@ -62,6 +60,7 @@ const demo = async (x) => {
 };
 
 (async () => {
-  await demo(1);
-  await demo(null);
+  const stdlib = await stdlib_loader.loadStdlib();
```
Comment (jeapostrophe) [inline examples/maybe-send/index.mjs:63]: That would make it so this file doesn't need to change

File: examples/variable-transfers/index.mjs [hunk 3]
```diff
@@ -41,7 +40,8 @@ export const run = async (n) => {
 };
 
 (async () => {
-  await run(0);
-  await run(1);
-  await run(2);
+  const stdlib = await loadStdlib();
```
Comment (jeapostrophe) [inline examples/variable-transfers/index.mjs:43]: Ditto

File: examples/workshop-trust-fund/index.mjs [hunk 4]
```diff
@@ -57,7 +56,8 @@ const runDemo = async (delayReceiver, delayFunder) => {
 };
 
 (async () => {
-  await runDemo(false, false);
-  await runDemo(true, false);
-  await runDemo(true, true);
+  const stdlib = await stdlib_loader.loadStdlib();
```
Comment (jeapostrophe) [inline examples/workshop-trust-fund/index.mjs:59]: Ditto

File: js/stdlib/ts/registry.ts [hunk 5]
```diff
@@ -0,0 +1,15 @@
+import { Connector } from "./ConnectorMode";
+
+let currentConnector : null | true | Connector = null;
+
+export const unsafeAllowMultipleStdlibs = () => {
+  currentConnector = true;
+};
+
+export const doStdlibLoad = (connector: Connector) => {
+  if (currentConnector == null || currentConnector == true) {
```
Comment (jeapostrophe) [inline js/stdlib/ts/registry.ts:10]: `currentConnector === connector`

**Review by jeapostrophe — CHANGES_REQUESTED** [review]

File: docs-src/ref-frontends-js.scrbl (hunk 1 above)
Comment (chrisnevers) [reply docs-src/ref-frontends-js.scrbl:119]: okay, sorry I misinterpreted the ticket

**Comment by jeapostrophe** [comment]
It's good to merge

**Pull request merged**